ops.cr

```diff
@@ -3,12 +3,15 @@ require "colorize"
 require "ops"
 
 def usage
-  STDERR.puts "Usage: ops [-f <filename>] <action>".colorize(:red)
+  STDERR.puts Builtins::Help.usage.colorize(:red)
   exit(1)
 end
 
 usage if ARGV.empty?
 
+# Enforce `--quiet` to act on a per-call basis.
+ENV.delete("OPS_QUIET_OUTPUT")
```

Comment:
So this env var can't be set to quiet any ops command that I run, it's just used by other crystal code as a flag of sorts?

So far, I don't think anything else in crops is doing this. ENV vars are for taking input from the environment. Behaviour-changing options are usually passed around as args.

I don't see another easy way to do this, though, so I'll create an issue to look into it and maybe follow up.

Reply:
Yeah true, that was pretty hacky on my part lol. In an attempt to not disturb what already is, I disturbed what already is!

Reply:
So what I ended up doing here was:
- Supporting the original _per-call_ nature.
- Having `$OPS_QUIET_OUTPUT` act as a global setting, in either direction; so:
  - `$OPS_QUIET_OUTPUT == "true"`
    - no `ops` commands (_nested or otherwise_) are printed to `stderr`.
  - `$OPS_QUIET_OUTPUT == "false"`
    - all `ops` commands (_nested or otherwise_) are printed to `stderr`.

All permutations accounted for in `spec/e2e/opt_q`.